.gitignore.txt

```diff
@@ -0,0 +1,22 @@
+# Compiled class file
+*.class
+
+# Log file
+*.log
+forIgnore/
+
+# BlueJ files
+*.ctxt
+
+# Mobile Tools for Java (J2ME)
```

Comment:
Please remove this section of ignoring (# Mobile Tools for Java (J2ME) and .mtj.tmp/) as for now it doesn't needed